package.json

```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "description": "A Fast Rust-based web bundler",
   "private": true,
+  "main": "./packages/rspack/dist/index.js",
```

Comment:
In some tests it tries to import ...

https://github.com/web-infra-dev/rspack/blob/d3ddbe7d2c3bca2995998a3365c375e19ce8eb7f/webpack-test/configCases/web/nonce/webpack.config.js#L1

This will correctly resolve it.